web/ui/react-app/src/types/PathPrefixProps.ts

```diff
@@ -1,5 +1,6 @@
 interface PathPrefixProps {
   pathPrefix?: string;
+  consolesLink?: string | null;
```

Comment:
@juliusv in response to the last part of this comment https://github.com/prometheus/prometheus/pull/6761#discussion_r375805037
Is there an option to be undefined? I guess the API will return null either way if something unexpected happens
I'm for making them ```string | null``` and to remove the optional operator (```?```)

Reply:
@boyskila Can you explain the particular problem in detail?

Reply:
Not sure whether I want to call it a problem....just thinking over what @juliusv mention in https://github.com/prometheus/prometheus/pull/6761#discussion_r375805037 about
```string | null``` vs ```?``` ie. ```string | undefined```

Reply:
And as I said I'm more for 
```suggestion
  pathPrefix: string | null;
  consolesLink: string | null;
```

Reply:
@boyskila We can acheive it by adding a new type file `ConsolesLinkType`
![Screenshot from 2019-12-12 04-33-29](https://user-images.githubusercontent.com/42701709/74018312-b30e6b00-49bb-11ea-89e8-6acaa798b447.png)
![Screenshot from 2020-02-07 14-56-49](https://user-images.githubusercontent.com/42701709/74018314-b43f9800-49bb-11ea-8afe-13c1a382be45.png)
![Screenshot from 2020-02-07 14-57-11](https://user-images.githubusercontent.com/42701709/74018315-b4d82e80-49bb-11ea-9394-391cf9d26098.png)
![Screenshot from 2020-02-07 14-57-28](https://user-images.githubusercontent.com/42701709/74018319-b570c500-49bb-11ea-9a03-c9c405b6aea3.png)
@juliusv @boyskila Any thoughts on this one

Reply:
Yep, `consolesLink?: string | null` would imply double optionality (it can be either `undefined` or `null`, not just one of them). I'd be for making it required, but nullable (coming from Go, I like things explicit :)).

Reply:
Not sure whether worth to split into separate interfaces. The point was just to type both variables explicitly to string | null. Btw, what about calling this interface ```GlobalProps``` or something. WDYT?

Reply:
@boyskila Seems our comments overlapped on that, I'd prefer https://github.com/prometheus/prometheus/pull/6761/files#r376299510 since I don't want anyone instantiating any of the unrelated components to also have to pass a completely unrelated `consolesLink` to all components that have a path prefix... like let's keep it localized to the one component that really needs it?